checker/tests/nullness/ConcurrentHashMapTest.java

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
By convention, when a test is about an issue in the issue tracker, we write a comment at the top of the file.  See other test files in the test suite.